### ISSUE TITLE
Capulettium lasts forever in xenomorphs

### ISSUE THREAD

**Issue by Katrina752** [ParadiseSS13/Paradise#21217]
**Issue Description**:

Capulettium appears to last forever in xenos. Was a xenomorph hunter, hit by a single capulettium syringe and I never woke up. Lasted 8 minutes before finally being killed.

**What did you expect to happen**: 

To wake up like normal after capulettium wears off.

**What happened instead**:

Didn't wake up.

**Why is this bad/What are the consequences**:

Allows cheesing with chemicals to easily perma disable xenos. A syringe gun with cap can single handedly wipe out the xenos if they hit them all.

**Steps to reproduce the problem**:

Make hunter.
Shoot hunter with syringe full of capulettium.
Watch it never wake up.

**When did the problem start happening**:

When shot with a capulettium syringe.

**Extra information**:

Sleepy.

**Comment by CinnamonSnowball** [comment]
Cannot reproduce; Capulettium knocks an alien hunter out only for the expected 2 minutes.

Do you think it might've been Capulettium Plus, which doesn't have a max duration, but can easily be woken up out of by simply un-resting?

**Comment by Katrina752** [comment]
Nope, I tried that. I was completely helpless the entire time. Nothing worked to the pointed I began to mhelp and ahelp before an astute crewmember weldered me to death.

**Comment by DGamerL** [comment]
Can replicate, shot a xenomorph with 15u of capulettium, they were out for at least 5 minutes, I stopped my testing afterwards.
